discord_bots/cogs/config.py

```diff
@@ -0,0 +1,118 @@
+            config = session.query(Config).first()
+            config.default_trueskill_mu = value
+            session.commit()
+
+        description = f"Default mu set to {value} by <@{interaction.user.id}>"
+        embed = Embed(description=description, colour=Colour.green())
+        await interaction.response.send_message(embed=embed)
+        if env_config.ADMIN_LOG_CHANNEL:
+            admin_log_channel = bot.get_channel(env_config.ADMIN_LOG_CHANNEL)
+            if isinstance(admin_log_channel, TextChannel):
+                await admin_log_channel.send(embed=embed)
+        return embed
+
+    @group.command(
+        name="setdefaultsigma", description="Set the default sigma for new players"
+    )
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    async def setdefaultsigma(self, interaction: Interaction, value: float):
+        """
+        Set the default sigma for new players
+        """
+        with Session() as session:
+            config = session.query(Config).first()
+            config.default_trueskill_sigma = value
+            session.commit()
+
+        description = f"Default sigma set to {value} by <@{interaction.user.id}>"
+        embed = Embed(description=description, colour=Colour.green())
+        await interaction.response.send_message(embed=embed)
+        if env_config.ADMIN_LOG_CHANNEL:
+            admin_log_channel = bot.get_channel(env_config.ADMIN_LOG_CHANNEL)
+            if isinstance(admin_log_channel, TextChannel):
+                await admin_log_channel.send(embed=embed)
+
+    @group.command(
+        name="setdefaulttau", description="Set the default tau for new players"
+    )
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    async def setdefaulttau(self, interaction: Interaction, value: float):
+        """
+        Set the default tau for new players
+        """
+        with Session() as session:
+            config = session.query(Config).first()
+            config.default_trueskill_tau = value
+            session.commit()
+
+        description = f"Default tau set to {value} by <@{interaction.user.id}>"
+        embed = Embed(description=description, colour=Colour.green())
+
+        await interaction.response.send_message(embed=embed)
+        if env_config.ADMIN_LOG_CHANNEL:
+            admin_log_channel = bot.get_channel(env_config.ADMIN_LOG_CHANNEL)
+            if isinstance(admin_log_channel, TextChannel):
+                await admin_log_channel.send(embed=embed)
+
+    @group.command(name="list", description="List current configuration values")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    async def listconfig(self, interaction: Interaction):
+        """
+        List current configuration values
+        """
+        with Session() as session:
+            config = session.query(Config).first()
+
+            embed = Embed(
```

Comment:
tbh i used ai to generate this, theres probably a more aesthetic way to display the config